include/sacn/receiver.h

```diff
@@ -93,13 +92,32 @@ typedef struct SacnRemoteSource
   char name[SACN_SOURCE_NAME_MAX_LEN];
 } SacnRemoteSource;
 
+/*! Information about a sACN source that was found. */
+typedef struct SacnFoundSource
+{
+  /*! The Source Description. */
+  SacnRemoteSource info;
+  /*! The address from which we received these initial packets. */
+  EtcPalSockAddr from_addr;
+  /*! The per-universe priority. */
+  uint8_t priority;
+  /*! The DMX (startcode 0) data. */
+  uint8_t values[DMX_ADDRESS_COUNT];
+  /*! The count of valid values. */
+  size_t values_len;
+  /*! Whether or not we only saw startcode 0 packets with the preview flag set. */
+  bool values_were_preview;
+  /*! The per-address priority (startcode 0xdd) data, if the source is sending it. */
+  uint8_t per_address[DMX_ADDRESS_COUNT];
+  /*! The count of valid priorities. */
+  size_t per_address_len;
+} SacnFoundSource;
+
 /*! Information about a sACN source that was lost. */
 typedef struct SacnLostSource
 {
-  /*! The Component Identifier (CID) of the source. */
-  EtcPalUuid cid;
-  /*! The name of the source. */
-  char name[SACN_SOURCE_NAME_MAX_LEN];
+  /*! The Source Description. */
+  SacnRemoteSource info;
```

Comment:
I understand the desire to reuse structs that are already defined, but I think this puts a little friction on the API user who now has to understand the layout of two structs in order to parse the callback.

Reply:
So it doesn't seem weird to you to have SacnRemoteSource, SacnFoundSource, and SacnLostSource all have cid and name members?  It tweaks my brain a little, but if you say you're sure you want it changed, I'll do it.

Reply:
Not really. They all represent sources. Sources always have CIDs and names.